plugins/autocertkit

```diff
@@ -450,8 +452,12 @@ def make_local_call(call, logging=True, std_out=subprocess.PIPE, shell=False):
         log.debug("make_local_call: %s" % (" ".join(call)))
     process = subprocess.Popen(call, stdout=std_out, shell=shell)
     stdout, stderr = process.communicate()
-    res = {"returncode": process.returncode, "stdout": str(
-        stdout).strip(), "stderr": str(stderr).strip()}
+    if stdout is not None:
+        res = {"returncode": process.returncode, "stdout": 
+            stdout.decode().strip(), "stderr": str(stderr).strip()}
+    else:
+        res = {"returncode": process.returncode, "stdout": 
+            str(stdout).strip(), "stderr": str(stderr).strip()}
```

Comment:
I am not sure if it could return `None` actually, while the code you should simplify: 
```{"returncode": process.returncode, "stdout": <Move if else here.>, "stderr": str(stderr).strip()}```